content/en/continuous_integration/intelligent_test_runner/javascript.md

```diff
@@ -27,32 +27,83 @@ Intelligent Test Runner is only supported in the following versions and testing
 * `cypress>=6.7.0`
   * From `dd-trace>=4.2.0`, `dd-trace>=3.23.0` or `dd-trace>=2.36.0`.
   * Instrument your web application with code coverage: see more in [Cypress setup](#cypress-setup)
+* If you're reporting data through the Datadog Agent, use v6.40+/v7.40+.
 
 ## Setup
 
+### Test Visibility Setup
+
 Prior to setting up Intelligent Test Runner, set up [Test Visibility for Javascript and Typescript][2]. If you are reporting data through the Agent, use v6.40+/v7.40+.
 
-To enable Intelligent Test Runner, set the following environment variables:
+### UI activation
+You or a user in your organization with "Intelligent Test Runner Activation" permissions must activate the Intelligent Test Runner on the [Test Service Settings][3] page.
+
+{{< img src="continuous_integration/itr_overview.png" alt="Intelligent test runner enabled in test service settings in the CI section of Datadog.">}}
+
+### Configuring the test runner environment
+
+{{< tabs >}}
+
+{{% tab "On-Premises CI Provider (Datadog Agent)" %}}
```

Comment:
Should we use shortcodes here like we do in https://github.com/DataDog/documentation/blob/master/content/en/continuous_integration/tests/python.md?plain=1#L41 ?

Our content is practically identical.

Reply:
yeah I think that's a good idea. I'll do it